server.js

```diff
@@ -72,7 +72,7 @@
     }
 
     // minify?
-    args.push(query.minify == 'true' ? '--minify' : '--debug')
+    args.push(query.minify == 'true' ? '--production' : '--debug')
```

Comment:
The regexp [here](https://github.com/mathiasbynens/lodash-webbuilder/blob/05a0b4671a29b89554113ccddd0d7cff9e87e334/server.js#L51) needs to be adjusted to remove `underscore|backbone`